lib/rspec-puppet-facts.rb

```diff
@@ -403,7 +403,7 @@ def self.facter_version_to_loose_requirement_string(version)
   def self.facter_version_for_puppet_version(puppet_version)
```

Comment:
I don't mean to add on all the time, but I'd like to make this consistent and mention OpenVox. How about
```suggestion
  def self.openfact_version_for_openvox_version(openvox_version)
```

And design wise: I really would appreciate a non-breaking release first (if possible). Then you may want to split out some version finding logic to a helper method.